docs/azure_arc_jumpstart/azure_arc_data/day2/aks/aks_mssql_dr/_index.md

```diff
@@ -0,0 +1,346 @@
+  
+  ![Screenshot showing disaster recovery configuration](./39.png)
+
+- Open Microsoft SQL Server Management Studio (SSMS) which is installed automatically for you as part of the bootstrap Jumpstart scenario and use the primary endpoint IP address for the primary cluster and login to the primary DB instance using the username and password provided in the text file mentioned above.
+
+  ![Screenshot showing opening SQL Server Management Studio from the start menu](./40.png)
+
+- Use the username and password you entered when provisioned the environment and select “SQL Server Authentication”. Alternatively, you can retrieve the username and password using the _`$env:AZDATA_USERNAME`_ and _`$env:AZDATA_PASSWORD`_ commands.
+
+  ![Screenshot showing logging into the SQL Server Management Studio](./41.png)
+
+- Connect to the secondary instance as well using the primary endpoint IP address for the secondary cluster in the in the text file mentioned above.
+
+  ![Screenshot showing the SQL Server Management Studio after login](./42.png)
+
+  ![Screenshot showing the SQL Server Management Studio after login](./43.png)
+
+  ![Screenshot showing SqlQueryStress connected](./44.png)
+
+- Expand the _Always On High Availability_ node on both instances to verify that the distributed availability group is created.
+
+  ![Screenshot showing SqlQueryStress connected](./45.png)
+
+- You will find the _AdventureWorks2019_ database already deployed into the primary instance (js-sql-pr) and automatically replicated to the secondary instance (js-sql-dr) as part of the distributed availability group.
+
+  ![Screenshot showing adventureworks database opened on the primary instance](./46.png)
+
+  > **NOTE: You will not be able to browse the _AdventureWorks2019_ database from the secondary instance since this instance is configured as a disaster recovery instace**.
+
+  ![Screenshot showing adventureworks database opened on the secondary instance](./47.png)
+
+### Simulating failure on the primary site
+
+- First, to test that the DB replication is working, a simple table modification is needed. For this example, on the primary replica, run the following query to update the title of one of the rows to be _Jumpstart Administrator_
+
+   ```shell
+    USE [AdventureWorks2019]
+    GO
+    UPDATE [HumanResources].[Employee]
+    SET [JobTitle] = 'Jumpstart Administrator'
+    WHERE NationalIDNumber = 245797967
+    GO
+    ```
+
+    ![Screenshot showing updating a record in the database](./48.png)
+
+    ![Screenshot showing the updated record in the database](./49.png)
+
+- To simulate a disaster situation, navigate to the AKS cluster in the Azure portal and stop the primary cluster.
+
+    ![Screenshot showing stopping the primary AKS cluster](./50.png)
+
+- Try to refresh the connection to the primary instance and you can see that its no longer available.
```

Comment:
how much until you will notice the change? might be good to add a note so the reader will know what to expect